frontend/src/org/components/EventForm/steps/registration/ShowApplicationModal.tsx

```diff
@@ -116,7 +102,7 @@ export const ShowApplicationModal: FC<IShowApplicationModalProps> = ({
       {currentApplication &&
         (!participantsMap ||
           !user ||
-          (user && currentApplication.id.toString() === user.id)) && (
+          (user && currentApplication.user?.toString() === user.id)) && (
```

Comment:
User by měl být `string` (nebo `null`), takže by nemuselo být nutné převádět typy. Narozdíl od id přihlášky, které je `number`.

Mimochodem, tady mě fascinuje autor původního kódu, kterému typescript asi hlásil, že ty typy nesedí, a místo toho, aby přemýšlel, jestli náhodou neporovnává špatné hodnoty, tak to prostě přetypoval. K čemu pak ten typescript je?